### ISSUE TITLE
Camel case components name doesn't load the props 

### ISSUE THREAD

**Issue by milad-alizadeh** [pocka/storybook-addon-vue-info#53]
Whenever I name my components as camel case the Props section in the info doesn't load.

For example: 
I have a component called `PlaceholderBox.vue`. The code below doesn't load the "props"

```
import PlaceholderBox from '../components/PlaceholderBox'
import VueInfoAddon from 'storybook-addon-vue-info'

storiesOf('PromoList', module)
  .addDecorator(VueInfoAddon)
  .add('default', () => ({
    component: { PlaceholderBox }
    template: `<PlaceholderBox />`
  }))
```

<img width="826" alt="screen shot 2018-08-17 at 15 52 11" src="https://user-images.githubusercontent.com/9379343/44272884-eabcac80-a235-11e8-90a6-aa27bfec5ace.png">

However if I change the template to `<placeholder-box />` then it loads the props

```
import PlaceholderBox from '../components/PlaceholderBox'
import VueInfoAddon from 'storybook-addon-vue-info'

storiesOf('PromoList', module)
  .addDecorator(VueInfoAddon)
  .add('default', () => ({
    component: { PlaceholderBox }
    template: `<placeholder-box />`
  }))
```

<img width="821" alt="screen shot 2018-08-17 at 15 53 57" src="https://user-images.githubusercontent.com/9379343/44272894-f0b28d80-a235-11e8-88ce-c5f97c4a9045.png">

Is this an expected behaviour? If so can we have support for camel case components names?

**Comment by pocka** [comment]
Fixed in v0.6.1

@milad-alizadeh Thank you for reporting!

**Comment by milad-alizadeh** [comment]
Great thanks!
